src/modules/shortcut_guide/overlay_window.cpp

```diff
@@ -32,12 +32,12 @@ namespace
         UNKNOWN,
         MINIMIZED,
         MAXIMIZED,
-        SNAPED_TOP_LEFT,
-        SNAPED_LEFT,
-        SNAPED_BOTTOM_LEFT,
-        SNAPED_TOP_RIGHT,
-        SNAPED_RIGHT,
-        SNAPED_BOTTOM_RIGHT,
+        SNAPPED_TOP_LEFT,
+        SNAPPED_LEFT,
+        SNAPPED_BOTTOM_LEFT,
+        SNAPPED_TOP_RIGHT,
+        SNAPPED_RIGHT,
+        SNAPPED_BOTTOM_RIGHT,
```

Comment:
Oh, I meant to flag this, this is a change to enums. Since they are defined locally w/in a `.cpp` file, it shouldn't be a problem.

Reply:
OK, thanks for pointing it out.